packages/mini-oxygen/src/vite/plugin.ts

```diff
@@ -86,52 +81,40 @@ export function oxygen(pluginOptions: OxygenPluginOptions = {}): Plugin[] {
           };
         },
       },
-      configureServer(viteDevServer) {
-        // For transform hook:
-        resolvedConfig = viteDevServer.config;
-
-        if (miniOxygen) return;
-        if (apiOptions.shouldStartRuntime?.() !== true) return;
-
-        const entry =
-          apiOptions.entry ?? pluginOptions.entry ?? DEFAULT_SSR_ENTRY;
-
-        absoluteWorkerEntryFile = path.isAbsolute(entry)
-          ? entry
-          : path.resolve(resolvedConfig.root, entry);
+      configureServer: {
+        order: 'pre',
```

Comment:
Adding `pre` here means we always run before Remix plugin, thus fixing the situation where the user adds `remix(), oxygen()` in the wrong order.